e2e/provision/hacks/vlan-interfaces.sh

```diff
@@ -21,7 +21,7 @@ source "$E2EDIR/defaults.env"
 kubeconfig="$HOME/.kube/config"
 
 for worker in $(kubectl get machines -l cluster.x-k8s.io/control-plane!= -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' --kubeconfig "$kubeconfig"); do
-    for i in {2..4}; do
+    for i in {2..6}; do
```

Comment:
Would adding a comment explain why the loop is `{2..6}'? Thanks

Reply:
@arora-sagar could you provide an answer here? My understanding is that OAI requires more VLAN IDs, is that right?

Reply:
@vjayaramrh Actually it is not OAI that requires more VLAN-ID it is the the scenario of R2 in which we have the O-RAN use-case in which we require more VLAN because of the RAN interfaces (E1 and F1).